apis/capabilities/v1beta1/policyregistry_types.go

```diff
@@ -32,70 +31,70 @@ import (
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 const (
-	PolicyKind = "Policy"
+	PolicyRegistryKind = "PolicyRegistry"
 
-	// PolicyInvalidConditionType represents that the combination of configuration
-	// in the PolicySpec is not supported. This is not a transient error, but
+	// PolicyRegistryInvalidConditionType represents that the combination of configuration
+	// in the PolicyRegistrySpec is not supported. This is not a transient error, but
 	// indicates a state that must be fixed before progress can be made.
-	PolicyInvalidConditionType common.ConditionType = "Invalid"
+	PolicyRegistryInvalidConditionType common.ConditionType = "Invalid"
```

Comment:
The `Policy` prefix in the conditions has been replaced by the `PolicyRegistry` prefix